Library/Homebrew/extend/object.rbi

```diff
@@ -0,0 +1,6 @@
+# typed: strict
+
+class Object
+  sig { returns(T::Boolean) }
+  def present?; end
```

Comment:
This is necessary for the return type of `PyPi::Package#valid_pypi_package?` to typecheck.